tests/python/pants_test/test_base.py

```diff
@@ -366,7 +366,6 @@ def pants_workdir(self):
   @classmethod
   @memoized_method
   def _build_root(cls):
-    cls.real_build_root = BuildRoot().path
     return os.path.realpath(mkdtemp(suffix='_BUILD_ROOT'))
```

Comment:
The code that was copying files into the workspace in `jvm_tool_task_test_base.py` was putting them inside this synthetic buildroot. But using `files(..)` targets won't do that: they'll end up directly in the CWD of the process. So this might need some more changes.